nodes/nomos-node-api/src/http/backend/axum.rs

```diff
@@ -18,10 +23,23 @@ use crate::{
     Backend,
 };
 
-#[derive(Clone)]
+/// Configuration for the Http Server
+#[derive(Debug, Clone, serde::Deserialize, serde::Serialize)]
+#[cfg_attr(feature = "clap", derive(clap::Args))]
 pub struct AxumBackendSettings {
-    pub addr: SocketAddr,
-    pub handle: OverwatchHandle,
+    /// Socket where the server will be listening on for incoming requests.
+    #[cfg_attr(feature = "clap", arg(
+        short, long = "http-addr",
+        default_value_t = std::net::SocketAddr::new(
+            std::net::IpAddr::V4(std::net::Ipv4Addr::new(127, 0, 0, 1)),
+            8080,
+        ),
+        env = "HTTP_BIND_ADDRESS"
+    ))]
+    pub address: std::net::SocketAddr,
+    /// Allowed origins for this server deployment requests.
+    #[cfg_attr(feature = "clap", arg(long = "http-cors-origin"))]
+    pub cors_origins: Vec<String>,
```

Comment:
Why do we have clap here? Are we using it as standalone somewhere?

Reply:
In the old HTTP service, we have clap, so I just keep them in the new HTTP service. I have not seen we use it standalone.

Reply:
I think we should remove it if we do not use it.